controllers/statusreport/statusreport_adapter.go

```diff
@@ -121,7 +121,9 @@ func (a *Adapter) EnsureSnapshotFinishedAllTests() (controller.OperationResult,
 	if !allIntegrationTestsFinished {
 		a.logger.Info("Not all required Integration PipelineRuns finished",
 			"snapshot.Name", a.snapshot.Name)
-		return controller.ContinueProcessing()
+		return controller.ContinueProcessing() // ---------> would we get stuck here indefintely because allIntegrationTestsFinished will
```

Comment:
Comments should be above the affected line, not bellow.

Should be this comment removed?

Reply:
sorry, this is a result of me not pushing my latest changes to the PR.